ISSUES.md

```diff
@@ -0,0 +1,225 @@
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# How To Request Support
+
+This is an Open Source Project so please be mindful that like in any other project of this kind there is no obligation to answer all requests for help. 
+
+This document's main purpose is to provide guidelines at how you can formulate your requests to increase your chances to be understood and to get supported.
+
+There are two main venues to receive support: [the forums](https://discuss.huggingface.co/) and [the github issues](https://github.com/huggingface/transformers/issues).
+
+## The Forums
+
+[The user forums](https://discuss.huggingface.co/) are supported by the wide community of the library users and backed up by developers when needed.
+
+If you have a difficulty with deploying this library or some questions, or you'd like to discuss a new feature, please first consider discussing those things at the forums. Only when you feel your subject matter has been crystalized and you still need support from the library developers do proceed to file an [Issue](https://github.com/huggingface/transformers/issues).
+
+In particular all "Please explain" questions or objectively very user-specific feature requests belong to the forums. Here are some example of such questions:
+
+* "I would like to use a BertModel within a RL-Agent for a customer support service. How can I use a BertForMaskedLM in my ChatBotModel?"
+
+* "Could you please explain why T5 has no positional embedding matrix under T5Model?"
+
+* "How should I set my generation parameters for translation?"
+
+* "How to train T5 on De->En translation?"
+
+
+## The github Issues
+
+Everything which hints at a bug should be opened as an [Issue](https://github.com/huggingface/transformers/issues).
+
+You are not required to read the following guidelines before opening an Issue. However, if you notice that your issue is not being addressed, chances are that the developers have one or several difficulties with the quality of that Issue. In which case please read the following guidelines and adjust your Issue accordingly.
+
+The guidelines:
+
+1. Before posting an Issue, first search for already posted issues, since chances are someone has already asked a similar question before you. 
+
+    If you use Google your search query should be:
+    
+    ```
+    "huggingface" "transformers" your query 
+    ```
+    
+    The first two quoted words tell Google to limit the search to the context of the Huggingface Transformers. The remainder is your query - most commonly this would be an assert line if the software fails with an assert. 
```

Comment:
Not sure assert line is going to help beginners a lot.
```suggestion
    The first two quoted words tell Google to limit the search to the context of the Huggingface Transformers. The remainder is your query - most commonly this would be a simplified version of your error message.
```

Reply:
I hear you. But I'm not sure how "a simplified version of your error message" is any more clear.

Perhaps leaving the original and using an example to show what is meant by an assert and how to convert it into a useful search query?

Reply:
e.g.:
```
If you have an error message like:
  ```python
   $ python -c 'import transformers'
   Traceback (most recent call last):
     File "<string>", line 1, in <module>
     File "/transformers/src/transformers/__init__.py", line 34, in <module>
       from . import dependency_versions_check
     File "/transformers/src/transformers/dependency_versions_check.py", line 34, in <module>
       from .file_utils import is_tokenizers_available
     File "/transformers/src/transformers/file_utils.py", line 40, in <module>
       from tqdm.auto import tqdm
   ModuleNotFoundError: No module named 'tqdm.auto'
   ```
   
   Search first for:
   1. search first for "huggingface" "transformers" "ModuleNotFoundError: No module named 'tqdm.auto'"
   2. if you don't find relevant results, then search for just "ModuleNotFoundError: No module named 'tqdm.auto'"
   3. and finally if nothing comes up remove the quotes

If the error includes any error messages that include bits of your filesystem, always remove those in the search query since other users will not have the same bits as yours. In such cases always remove double quotes since your query is not longer exact. For example:

python -c 'open("/tmp/wrong_path.txt", "r")'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/wrong_path.txt'

Here you'd search for just: `FileNotFoundError: [Errno 2] No such file or directory`

```

Reply:
Yes showing an example is great!

Reply:
Done. I also rewrote the bit on assert line and explained what it means.